src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/RunnerService.kt

```diff
@@ -13,28 +17,68 @@ class RunnerService(private val project: Project) {
     /**
      * Method to show notification that test generation is already running.
      */
-    private fun showNotification() {
+    private fun showGenerationRunningNotification() {
         NotificationGroupManager.getInstance()
             .getNotificationGroup("EvoSuite Execution Error")
             .createNotification(
-                "EvoSuite is already running",
-                "You can generate more tests after the current test generation is complete.",
+                TestGenieBundle.message("alreadyRunningNotificationTitle"),
+                TestGenieBundle.message("alreadyRunningTextNotificationText"),
                 NotificationType.WARNING
             )
             .notify(project)
     }
 
     /**
-     * Method to verify whether EvoSuite is running or not.
+     * Method to show notification that the class cannot be parsed.
+     */
+    private fun showParsingFailedNotification() {
+        NotificationGroupManager.getInstance()
+            .getNotificationGroup("EvoSuite Execution Error")
+            .createNotification(
+                TestGenieBundle.message("parsingFailedNotificationTitle"),
+                TestGenieBundle.message("parsingFailedNotificationText"),
+                NotificationType.ERROR
+            )
+            .notify(project)
+    }
+
+    /**
+     * Check if EvoSuite is running.
      *
-     * @return true if EvoSuite is currently running
+     * @return true if it is already running
      */
-    fun verifyIsRunning(): Boolean {
+    private fun isEvoSuiteRunning(): Boolean {
         if (isRunning) {
-            showNotification()
+            showGenerationRunningNotification()
             return true
         }
         isRunning = true
         return false
     }
+
+    /**
+     * Check if class is parsable.
+     *
+     * @return true if it is parsable
+     */
+    private fun isParsing(psiFile: PsiFile): Boolean {
```

Comment:
Great simple solution :)
My machine gets the correct notification when I mess up syntax.